### PR TITLE
Swipe option menu

### LINKED ISSUE #22: Timeline list item slide menu options

### PR DESCRIPTION
**Pull request by ravishrajput**

### CODE REVIEW

File: app/src/main/java/com/remindme/app/screens/timeline/TimelineListGroupView.kt [hunk 1]
```diff
@@ -143,8 +148,21 @@ fun TimelineListGroupView(list: List<Reminder>, timelineState: TimelineState) {
...
           }) {
-          list.forEach {
-            ReminderItemView(it)
+          list.forEach { item ->
+            RevealSwipe(
+              directions = setOf(
+                RevealDirection.End
+              ),
+              onContentClick = {
+                println("hello")
```
Comment (parmeshtoyou) [inline app/src/main/java/com/remindme/app/screens/timeline/TimelineListGroupView.kt]: it's better to go with Log.d :)

File: app/src/main/java/com/remindme/app/ui/component/SwipeRevealView.kt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  onDelete: () -> Unit
+) {
+  Row {
+    IconButton(
+      onClick = onEdit,
+      content = {
+        Icon(
+          painter = painterResource(id = R.drawable.ic_edit),
+          tint = Blue700,
+          contentDescription = "edit action",
```
Comment (parmeshtoyou) [inline app/src/main/java/com/remindme/app/ui/component/SwipeRevealView.kt]: it's better to refer string from string.xml for content description.

**Review by parmeshtoyou — COMMENTED** [review]
please check provided comments.

File: app/src/main/java/com/remindme/app/screens/timeline/TimelineListGroupView.kt (hunk 1 above)
Comment (ravishrajput) [reply app/src/main/java/com/remindme/app/screens/timeline/TimelineListGroupView.kt]: my bad I wanted to remove this.

**Review by parmeshtoyou — APPROVED** [review]

**Pull request merged**